formValidation.js

```diff
@@ -0,0 +1,32 @@
+(function(){
+    const dataTypesRegExp = {
+        address: /^0x[0-9a-f]{40}$/i
+    };
+
+    window.formValidation = {
+        validateArgsByEvent: validateArgsByEvent,
+        dataTypesRegExp: dataTypesRegExp
+    };
+
+    function validateArgsByEvent(args, methodName) {
+        let resultPromise = Promise.resolve([]);
+
+        args.forEach((argNode) => {
+            resultPromise = resultPromise.then(validateArg.bind(this, argNode, methodName));
```

Comment:
Consider passing `null` as a context if you do not care about it. In order to make it readable.

Reply:
ok, agree